hr_timesheet_current/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Victor Champonnois
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Hr Timesheet Current',
```

Comment:
```suggestion
    'name': 'My Current Timesheet Sheet',
```
or any other better-sounding name. ☺

Reply:
or, actually, just “Current Timesheet Sheet” would already be enough. but the choice is yours!